test/jasmine/tests/parcoords_test.js

```diff
@@ -0,0 +1,821 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+var mock0 = { // mock with zero dimensions; special case, as no dimension can be rendered
```

Comment:
No. I suggesting you should try to get image export working in the browser first.